about.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="about-header">
+    <nav class="social-media-nav">
+      <ul class="social-media">
+        <li class="social-list">
+          <a href="#"><img src="images/facebook (1).png" alt="facebook logo"></a>
+        </li>
+        <li class="social-list"><a href="#"><img src="images/twitter.png" alt="twitter logo"></a>
+        </li>
+        <li class="social-list">Spanish</li>
+        <li class="social-list">My Page</li>
+        <li class="social-list">logout</li>
+      </ul>
+    </nav>
+    <nav class="mobile-menu">
+      <div>
+        <a href="#"><img src="images/mobile_menu.png" alt="mobile menu" width="20" height="20"></a>
+      </div>
+    </nav>
+    <nav class="desktop-menu">
+      <a href="index.html"><img src="images/logo-en.svg" alt="logo culture days" width="100" height="64"></a>
+      <ul class="main-menu">
+        <li class="menu-list"><a href="about.html">About</a></li>
+        <li class="menu-list"><a href="index.html#program">Program</a></li>
+        <li class="menu-list"><a href="index.html#join">Join</a></li>
+        <li class="menu-list"><a href="index.html#sponsor">Sponsor</a></li>
+        <li class="menu-list"><a href="index.html#news">News</a></li>
+        <li class="campaign-menu"><a href="index.html">Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
```

Comment:
-- [ **OPTIONAL** ] I think it would be better if you replace the above navigation options with the ones containing your **About** page to make it more professional and easy to be navigated through. Right now, any of them can not redirect the user to one specific section of the page. I suggest you replace them with the sections' `id`s of the **About** page such as **Creative Culture Days Logo as '"Detail", past Culture Days,...**

- [ ] Kindly make sure the `header` design of the about page sticks matches the one from the template and the **About navigation bar** contains an option that allows the user to go back to the **Home** page for both the **Desktop version** and  **Mobile version**. Make sure you fix it.

![Screenshot from 2022-09-22 20-17-29](https://user-images.githubusercontent.com/83042742/191823028-1dc60aa2-43ff-4a57-add5-cd1ff7f7f91f.png)